client/src/locales/de.ts

```diff
@@ -67,8 +91,8 @@ export default {
 			description2:
 				"Beteiligen Sie sich an der Entwicklung, indem Sie Ihre Ideen oder Ihren Code beisteuern, oder zeigen Sie Ihre Unterstützung, indem Sie Spender werden. Alle Spenden werden verwendet, um die Betriebskosten und die Entwicklung von OpenTogetherTube zu finanzieren und um OpenTogetherTube werbefrei zu halten.",
 			how: "Wie kann ich helfen?",
-			sponsor: "@:nav.support",
```

Comment:
This is intentionally a reference. Does this change make more sense contextually where it appears than the referenced text?